### ISSUE TITLE
Refinement to holding "Ctrl" for dynamics assignment

### ISSUE THREAD

**Issue by avvvvve** [musescore/MuseScore#23956]
Follow up from @oktophonie's: https://github.com/musescore/MuseScore/issues/23321

> Adding a dynamic from the palette with the 'Ctrl' pressed should assign the dynamic(s) directly to the voice(s) of the currently selected note(s), if any, rather than to 'All voices'.

This assumes that Preferences > Note input > Voice assignment is set to "All voices on the instrument"

If it's set to "Only the voice they are applied to", then when adding a dynamic from the palette with 'Ctrl' pressed, the dynamic should be assigned to "All" voices (or "All voices on instrument" on grand staves). Right now, holding 'Ctrl' in that scenario does nothing.

Basically, 'Ctrl' should do the opposite of whatever the preference is set to.

<img width="330" alt="image" src="https://github.com/user-attachments/assets/3547fcef-28d6-41b0-829e-3d4bf1d5c25a">
